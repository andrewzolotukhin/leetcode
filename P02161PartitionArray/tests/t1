7 10
9 12 5 10 14 3 10
