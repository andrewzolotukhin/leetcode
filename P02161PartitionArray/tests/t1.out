9 5 3 10 10 12 14 
